File: java/set.java
```java
import java.util.ArrayList;
import java.util.Arrays;
import java.util.Collections;

public class Main {
    //并集
    public static ArrayList<Integer> Union(ArrayList<Integer> A, ArrayList<Integer> B) {
        ArrayList<Integer> result = new ArrayList<>(A);
        for (int b : B) {
            if (!A.contains(b)) {//如果集合B中元素不在集合A中
                result.add(b);
            }
        }
        Collections.sort(result);//排序
        return result;
    }

    //交集
    public static ArrayList<Integer> Intersection(ArrayList<Integer> A, ArrayList<Integer> B) {
        ArrayList<Integer> result = new ArrayList<>();
        for (int a : A) {
            if (B.contains(a)) {//如果集合A中元素在集合B中
                result.add(a);
            }
        }
        return result;
    }

    //差集A-B
    public static ArrayList<Integer> Difference(ArrayList<Integer> A, ArrayList<Integer> B) {
        ArrayList<Integer> result = new ArrayList<>();
        for (int a : A) {
            if (!B.contains(a)) {//如果集合A中元素不在在集合B中
                result.add(a);
            }
        }
        return result;
    }

    //补集
    public static ArrayList<Integer> Complement(ArrayList<Integer> A, ArrayList<Integer> U) {
        ArrayList<Integer> result = new ArrayList<>();
        for (int u : U) {
            if (!A.contains(u)) {//如果全集U中元素不在集合A中
                result.add(u);
            }
        }
        return result;
    }

    //对称差集
    public static ArrayList<Integer> Symmetric(ArrayList<Integer> A, ArrayList<Integer> B) {
        ArrayList<Integer> set1 = Difference(A, B);//A-B
        ArrayList<Integer> set2 = Difference(B, A);//B-A
        ArrayList<Integer> result = Union(set1, set2);//(A-B)U(B-A)
        Collections.sort(result);//排序
        return result;
    }

    public static void main(String[] args) {
        ArrayList<Integer> A = new ArrayList<>(Arrays.asList(1, 3, 4));
        ArrayList<Integer> B = new ArrayList<>(Arrays.asList(1, 2, 5, 6));
        ArrayList<Integer> U = new ArrayList<>(Arrays.asList(1, 2, 3, 4, 5, 6));

        ArrayList<Integer> union = Union(A, B);
        System.out.printf("集合A与集合B的并集：%s\n", union);

        ArrayList<Integer> intersection = Intersection(A, B);
        System.out.printf("集合A与集合B的交集：%s\n", intersection);

        ArrayList<Integer> difference = Difference(A, B);
        System.out.printf("集合A与集合B的差集：%s\n", difference);

        ArrayList<Integer> symmetric = Symmetric(A, B);
        System.out.printf("集合A与集合B的对称差集：%s\n", symmetric);

        ArrayList<Integer> complement = Complement(A, U);
        System.out.printf("集合A在全集U中的补集：%s\n", complement);
    }
}
```

File: python/set.py
```py
from typing import List  
  
class SetOperations:  
      
    # 并集  
    def union(self, A: List[int], B: List[int]) -> List[int]:  
        result = A.copy()  
        for b in B:  
            if b not in A:  
                result.append(b)  
        result.sort()  
        return result  
      
    # 交集  
    def intersection(self, A: List[int], B: List[int]) -> List[int]:  
        result = []  
        for a in A:  
            if a in B:  
                result.append(a)  
        return result  
      
    # 差集 A-B  
    def difference(self, A: List[int], B: List[int]) -> List[int]:  
        result = []  
        for a in A:  
            if a not in B:  
                result.append(a)  
        return result  
      
    # 补集  
    def complement(self, A: List[int], U: List[int]) -> List[int]:  
        result = []  
        for u in U:  
            if u not in A:  
                result.append(u)  
        return result  
      
    # 对称差集  
    def symmetric(self, A: List[int], B: List[int]) -> List[int]:  
        set1 = self.difference(A, B)  # A-B  
        set2 = self.difference(B, A)  # B-A  
        result = self.union(set1, set2)  # (A-B)U(B-A)  
        result.sort()  
        return result  
  
# 示例用法  
if __name__ == "__main__":  
    set_ops = SetOperations()  
    A = [1, 2, 3, 4]  
    B = [3, 4, 5, 6]  
    U = [1, 2, 3, 4, 5, 6, 7, 8]  
  
    print("Union:", set_ops.union(A, B))  
    print("Intersection:", set_ops.intersection(A, B))  
    print("Difference A-B:", set_ops.difference(A, B))  
    print("Complement:", set_ops.complement(A, U))  
    print("Symmetric Difference:", set_ops.symmetric(A, B))
```
